index.html

```diff
@@ -0,0 +1,167 @@
+        <div class="Home-Back">
+          <h2 class="Home-1-h2">"Hello! Playstation world"</h2>
+          <h1>
+            PLAYSTATION & Insomniac Games<br />
+            GAMES SHOWCASE<br />
+            2022
+          </h1>
+          <p>
+            Play brand new games the same day they release and hundreds of
+            additional high-quality games with Playstation Game Pass. Join us as
+            we showcase even more additions to your game pass catalog.
+          </p>
+          <h2>2021.10.27(TUE) ~ 28(WED)</h2>
+          <p>@ Weybridge KT13 0XW, United Kingdom</p>
+        </div>
+      </section>
+    </div>
+    <section class="main-page" id="Main-Program">
+      <h1 class="main-page-1">Main Program</h1>
+      <div class="border"></div>
+      <!-- Card 1 starting -->
+      <div class="card margin-l">
+        <div class="card-1 left">
+          <img src="./Img/Card-Icon/launch.png" alt="icon of rocket" />
+          <h5>launch</h5>
+          <p>
+            Show opener introductions with Erin Ashely and Malik. Secial look at
+            fans celebration of the xbox games showcase
+          </p>
+        </div>
+        <!-- Card -1 end -->
+        <!-- Card 2 starting -->
+        <div class="card-1 left card-im">
+          <img
+            src="./Img/Card-Icon/playstation-png.png"
+            alt="icon of Playstation"
+          />
+          <h5>Playstation</h5>
+          <p>
+            Console exclusive games showcase. This includes PS5 new version
+            console exclusive trailer.
+          </p>
+        </div>
+        <!-- Card 2 end -->
+        <!-- Card 3 starting -->
+        <div class="card-1 left card-im">
+          <img
+            src="./Img/Card-Icon/ps-controler.png"
+            alt="icon of Playstation controler"
+          />
+          <h5>Controler</h5>
+          <p>
+            Console exclusive games showcase. This includes new Controler 
+            exclusive trailer.
+          </p>
+        </div>
+        <!-- Card 3 end -->
+        <!-- Card 4 starting -->
+        <div class="card-1 left">
+          <img src="./Img/Card-Icon/behind-the-scenes.png" alt="icon of Behind the scenes" />
+          <h5>Behind the scenes</h5>
+          <p>
+            Behind the scenes:-  Create a game like new relase game Spiderman 
+            next year relaese game god of war. 
+          </p>
+        </div>
+        <!-- Card 4 end -->
+        <!-- Card 5 starting -->
+        <div class="card-1">
+          <img src="./Img/Card-Icon/peakpx.jpg" alt="icon of rocket" />
+          <h5>launch Future</h5>
+          <p>
+            Showcase the game of GOD OF WAY RAGNAROK. The new exlusive the 
+            game of Playstation.
+          </p>
+        </div>
+        <!-- Card 5 end -->
+      </div>
+      <button><a href="https://www.youtube.com/watch?v=T5VtfD2i3rc">Watch Playstation Showcase</a></button>
+    </section>
+    <section class="Featured" id="Featured"></section>
```

Comment:
Kindly adjust the speakers section in order to match the design.

See the illustrations in the image for the adjustment to be made:

Your design:
![image](https://user-images.githubusercontent.com/20554583/203769672-044ff994-360a-4cbb-9224-3fa1409b49d8.png)


Cindy's design:
![image](https://user-images.githubusercontent.com/20554583/203769808-d11e94fe-47e1-42ec-b3ed-f5768f60a7c2.png)